frontend/src/views/Contributors.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  div
+    h1 Contributors
+    h2(v-if="this.errorMessage") {{ errorMessage }}
+    div(v-for="contributor in contributors" class="result")
+      table(style='width:100%')
+        tr
+          th Name
+          th Github
+          th Portfolio
+          th LinkedIn
+        tr
+          td {{ contributor.name }}
+          td 
+            a(:href="contributor.html_url") {{contributor.login}}
+          td 
+            a(:href="contributor.portfolio") {{ contributor.portfolio && 'portfolio' }}
+          td
+            a(:href="contributor.linkedin") {{ contributor.linkedin && 'linkedin' }}
+
+</template>
+
+<style scoped>
+.result {
+  padding: 5px;
+  margin-bottom: 16px;
+}
```

Comment:
See frontend/src/styles/_dimensions.scss
use `vstep` to maintain vertical rhythm and `hstep` to maintain horizontal rhythm. You will probably have to `@import` these
Try the following values:
```scss
@import '../styles/dimensions';

.result {
  padding: vstep(1) hstep(1);
  margin-bottom: hstep(2);
}
```

Reply:
Ok. Thanks.

Reply:
I really liked this pug template system.